src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Digest.cs

```diff
@@ -16,27 +16,17 @@ internal static partial class AppleCrypto
         [DllImport(Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_DigestCreate")]
         internal static extern SafeDigestCtxHandle DigestCreate(PAL_HashAlgorithm algorithm, out int cbDigest);
 
-        internal static unsafe int DigestUpdate(SafeDigestCtxHandle ctx, ReadOnlySpan<byte> pbData, int cbData)
-        {
-            fixed (byte* pbDataPtr = &pbData.DangerousGetPinnableReference())
-            {
-                return DigestUpdate(ctx, pbDataPtr, cbData);
-            }
-        }
+        internal static int DigestUpdate(SafeDigestCtxHandle ctx, ReadOnlySpan<byte> pbData, int cbData) =>
+            DigestUpdate(ctx, ref pbData.DangerousGetPinnableReference(), cbData);
```

Comment:
If `cbData` is always equal to `pbData.Length`, you could change this (and similar places) to

```csharp
internal static int DigestUpdate(SafeDigestCtxHandle ctx, ReadOnlySpan<byte> pbData) =>
    DigestUpdate(ctx, ref pbData.DangerousGetPinnableReference(), pbData.Length);
```